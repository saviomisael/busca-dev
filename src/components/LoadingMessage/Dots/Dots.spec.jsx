import { Dots } from '.';

describe('<Dots />', () => {
  it('should matches with snapshot', () => {
    const { container } = global.renderTheme(<Dots />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
