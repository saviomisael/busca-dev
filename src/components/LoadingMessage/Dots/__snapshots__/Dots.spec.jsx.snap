// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dots /> should matches with snapshot 1`] = `
.c0 {
  display: inline-block;
  background: linear-gradient(#F5F5F5,#284B63);
  background-clip: text;
  -webkit-background-clip: text;
  color: transparent;
  font-size: 8.5rem;
}

.c0:first-child {
  -webkit-animation: ilFKjG 2s ease-in-out infinite;
  animation: ilFKjG 2s ease-in-out infinite;
}

.c0:nth-child(2) {
  -webkit-animation: ilFKjG 2s 0.1s ease-in-out infinite;
  animation: ilFKjG 2s 0.1s ease-in-out infinite;
}

.c0:nth-child(3) {
  -webkit-animation: ilFKjG 2s 0.2s ease-in-out infinite;
  animation: ilFKjG 2s 0.2s ease-in-out infinite;
}

<span
  class="c0"
>
  Â·
</span>
`;
